docs/source/compile/inductor_profiling.rst

```diff
@@ -0,0 +1,162 @@
+This document lists some useful commands and workflows that can help
+people dive into a model’s perf in TorchInductor. When a model is not
+running as fast as expected, we may want to dive into the model and
+check individual kernels. Usually those kernels taking the majority of
+GPU time are the most interesting ones. Once we decide the kernels, we
+may also want to run individual kernels directly and inspect its perf.
+We have tools to cover everything mentioned above.
+
+Relevant Environment Variables
+==============================
+
+-  TORCHINDUCTOR_UNIQUE_KERNEL_NAMES
+
+   -  By default, inductor names a triton kernel as ‘triton\_’. When
+      this envvar is enabled, inductor generates a more meaningful
+      kernel name in the trace, like ``triton_poi_fused_cat_155`` which
+      contains the kernel category (poi for pointwise) and original aten
+      ops. This config is disabled by default to improve the chance of
+      compilation cache hit.
+
+-  TORCHINDUCTOR_BENCHMARK_KERNEL
+
+   -  Enabling this will make inductor codegen harness to benchmark
+      individual triton kernels.
+
+-  TORCHINDUCTOR_MAX_AUTOTUNE
+
+   -  Inductor autotuner will benchmark more triton.Configs and pick the
+      one with the best perf. This will increase compilation time with
+      the hope to improve perf.
+
+Breakdown Model GPU Time
+========================
+
+Below are the steps to breakdown execution time of a model into
+individual kernels. We take mixnet_l as an example.
+
+1. Run the benchmark script for the model:
+
+TORCHINDUCTOR_UNIQUE_KERNEL_NAMES=1 TORCHINDUCTOR_BENCHMARK_KERNEL=1
+python -u benchmarks/dynamo/timm_models.py –backend inductor –amp
+–performance –dashboard –only mixnet_l –disable-cudagraphs –training
+
+NOTE: the tool relies on kernel name to decide its category. Enabling
+TORCHINDUCTOR_UNIQUE_KERNEL_NAMES is crucial for that.
+
+2. In the output log, look for lines like: **Compiled module path:
+   /tmp/torchinductor_shunting/qz/cqz7hvhood7y3psp7fy6msjxsxyli7qiwiybizdwtjw6ffyq5wwd.py**
+
+We have one line for each compiled module. If there are no extra graph
+breaks, we would see 2 such lines in the log, one for the forward graph
+and one for the backward graph.
+
+For our example command, we get the following compiled module for the
+forward and backward graphs respectively:
+
+-  https://gist.github.com/shunting314/c2a4d8a28b00fcb5586d0e9d9bf77f9f
+-  https://gist.github.com/shunting314/48efc83b12ec3ead950052e4a0220b10
+
+3. Now we can dive into the perf for each individual compiled module.
+   Let’s pick the one for the forward graph for illustration purposes.
+   I’ll name it fwd.py for convenience. Run it directly with ‘-p’
+   argument:
+
+**> python fwd.py -p**
+
+Here is the full output log I get:
+https://gist.github.com/shunting314/8243734a38b5733ea78479209c0ae893
+
+There are quite a lot of interesting things to note in the output
+
+1. We write a chrome trace file for the profile so we can load the trace
```

Comment:
Not very familiar with RST. But can we use a/b/c/d to mark list items here to avoid confusing with the previous list using 1/2/3 to mark list items. (similar to what we do in the original gdoc)